fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/OnlineIndexer.java

```diff
@@ -726,11 +725,11 @@ protected void buildIndexSingleTarget() {
      *
      * @param minSplit not split if it cannot be split into at least <code>minSplit</code> ranges
      * @param maxSplit the maximum number of splits generated
-     * @return a list of split primary key ranges (the low endpoint is inclusive and the high endpoint is exclusive)
+     * @return a list of split primary key ranges
      */
     @API(API.Status.EXPERIMENTAL)
     @Nonnull
-    public List<Pair<Tuple, Tuple>> splitIndexBuildRange(int minSplit, int maxSplit) {
+    public List<TupleRange> splitIndexBuildRange(int minSplit, int maxSplit) {
         return getIndexerByRecordsOrThrow().splitIndexBuildRange(minSplit, maxSplit);
```

Comment:
Since the API is changing in this PR, it might be a good idea to preemptively mark this function as deprecated. 
Letting users adjust their code for this change without knowing that this function is a candidate for removal (https://github.com/FoundationDB/fdb-record-layer/pull/2262) seems a little unfair.

Reply:
I've gone ahead and marked these methods as deprecated. I'll have to admit that #2262 sort of slipped my mind. It may be that we should just remove these methods now, if we're happy with mutual indexing, though I'm actually a bit out of touch with where we were with that project and with #2262 in particular